test/unit/spec/PasswordExpired_spec.js

```diff
@@ -87,17 +90,17 @@ function (Okta, OktaAuth, LoginUtil, Util, PasswordExpiredForm, Beacon, Expect,
     test.form.submit();
   }
 
-  function setupExcludeAttributes (excludeAttributesArray) {
+  function setupExcludeAttributes (excludeAttributesArray, showPasswordRequirementsAsHtmlList = false) {
     var passwordExpiredResponse = deepClone(resPassExpired);
     var policyComplexity = passwordExpiredResponse.response._embedded.policy.complexity;
     policyComplexity.excludeAttributes = excludeAttributesArray;
-    return setup(undefined, passwordExpiredResponse);
+    return setup({ 'features.showPasswordRequirementsAsHtmlList': showPasswordRequirementsAsHtmlList }, passwordExpiredResponse, null);
   }
 
   Expect.describe('PasswordExpiration', function () {
 
     Expect.describe('PasswordExpired', function () {
-
+      /* eslint max-statements: [2, 26] */
```

Comment:
you shall not need this override as `test/unit/.eslintrc.js` has done that.

Reply:
Hmm, my eslint kept failing though.

Reply:
IDE plugin bug? it shall work well from CLI.

Reply:
It was actually the CLI that showed me this error.

Reply:
Turns out there's a max-statements on the top of this file. Removing both since test/unit/.eslintrc.js has done that. Thanks @haishengwu-okta !